backend/src/main/kotlin/de/neuefische/backend/user/UserProfileResponse.kt

```diff
@@ -0,0 +1,9 @@
+package de.neuefische.backend.user
+
+data class UserProfileResponse(
+    val id: String,
+    val name: String,
+    val bio: String?,
+    val imageUrl: String?,
+    val mediums: List<String>
```

Comment:
```suggestion
    val mediums: List<Medium> // use actual enum
      
    // or make sure to avoid null:
    val mediums: List<String> = emptyList() 
```